source/js/buyers-guide/search.js

```diff
@@ -15,10 +15,54 @@ const categoryTitle = document.querySelector(`.category-title`);
 const parentTitle = document.querySelector(`.parent-title`);
 const toggle = document.querySelector(`#product-filter-pni-toggle`);
 const subcategories = document.querySelectorAll(`.subcategories`);
+const subContainer = document.querySelector(`.subcategory-header`);
 
 // TODO: turn this into a static class rather than plain JS object.
 const SearchFilter = {
   init: () => {
+    let pos = { top: 0, left: 0, x: 0, y: 0 };
+
+    const mouseDownHandler = function (event) {
```

Comment:
rather than `mouse down`, let's name this based on its functionality (the fact that we tie it to a mousedown is basically an irrelevant detail for the function's code =).

E.g `markScrollStart` or something?

Also, `const` makes sense for arrow functions, but is unnecessary for regular functions, which already get locally scoped by name. So if this function doesn't technically rely on any scoped vars: let's move it out `init`, it'll just be a plain function. If it does, let's make it an arrow function.